### PR TITLE
Version 2.1.0

### PR DESCRIPTION
**Pull request by ianardee**
# Version 2.1.0

## Description

* :sparkles: update to Invoices v3
* :recycle: refactor `Endpoint` classes and document building
* :arrow_up: Update PikePDF to 5.0.1
* :sparkles: add a basic logger

## How Has This Been Tested

Refactored and updated unit tests.

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Mindee Python API Helper Changelog
 
+## v2.1.0 (2022-03-02)
+
+### Changes
+
+* :sparkles: update to Invoices v3
```
Comment (fharper) [inline CHANGELOG.md:7]: Invoices API v3 maybe?

**Review by fharper — COMMENTED** [review]
We should add information about the logging in the README

File: CHANGELOG.md (hunk 1 above)
Comment (ianardee) [reply CHANGELOG.md]: done

**Comment by ianardee** [comment]
> We should add information about the logging in the README

@fharper we changed the readme recently to be very minimal, only the basic stuff to get going.

This is not at all required for using the lib, it should go in the docs.

**Comment by fharper** [comment]
> > We should add information about the logging in the README
> 
> @fharper we changed the readme recently to be very minimal, only the basic stuff to get going.
> 
> This is not at all required for using the lib, it should go in the docs.

Good point, yes, in the doc :)

**Review by fharper — APPROVED** [review]
LGTM

File: mindee/logger.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+"""
+Set up a logger.
+"""
+
+import os
+import logging
+
+LOGLEVEL = os.environ.get("MINDEE_LOGLEVEL", "INFO").upper()
```
[the remark this answers is not in the document]
Comment (ianardee) [reply mindee/logger.py:8]: Nor should it, this is the expected behavior.

[the remark this answers is not in the document]
Comment (ianardee) [reply mindee/logger.py:8]: This **is** the proper message, the exception is coming from the built-in Python logger.

**Review by jonathanMindee — COMMENTED** [review]
:100:

**Pull request merged**